contracts/foundry.toml

```diff
@@ -104,6 +104,12 @@ test = 'src/v0.8/workflow/test'
 via_ir = true # reconsider using the --via-ir flag if compilation takes too long
 evm_version = 'paris'
 
+[profile.data-feeds]
+optimizer_runs = 10_000
+src = 'src/v0.8/data-feeds'
+test = 'src/v0.8/data-feeds/test'
+solc_version = '0.8.24'
```

Comment:
Does it make sense to use 26 instead?

Reply:
why not 28 ? it was published 4 months ago

Reply:
CCIP is currently the highest at 26, there's no feature we thought was worth the upgrade. Doesn't mean we shouldn't use 28 but it didnt bring us any meaningful improvement.